src/ts/components/banners/cookie-banner.tsx

```diff
@@ -23,20 +23,30 @@ export type CookieBannerProps = {
    * @default 'bottom'
    */
   position?: 'top' | 'bottom';
+  /**
+   * Sets maximum age for the cookie in milliseconds
+   * @default 60 * 60 * 24 * 365
+   */
+  maxAge?: number;
 } & OptionalComponentPropAndHTMLAttributes;
 
 /**
  * A [Banner](#banner) component that is permanently dismissed after setting a cookie.
  * This component takes a render prop, which can be a component or function, that is passed a dismiss prop
  * which you can then apply as an onClick prop to an element of your choice.
  */
+
+const A_YEAR_IN_MS = 60 * 60 * 24 * 365;
```

Comment:
Isn't `60 * 60 * 24 * 365` a year in seconds, not milliseconds? From the docs it looks like it _should_ be in seconds, so I think the name is wrong rather than the calculation.